projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/vdk_jupyterlab_extension/handlers.py

```diff
@@ -89,7 +90,7 @@ def post(self):
             status = VdkUI.download_job(
                 input_data[VdkOption.NAME.value],
                 input_data[VdkOption.TEAM.value],
-                input_data[VdkOption.REST_API_URL.value],
+                os.environ["REST_API_URL"],
```

Comment:
Can you not make this a parameter of the methods and instead only load it where it is needed? 
It will make it clearer that it is a global value.